app/services/concerns/xccdf/profiles.rb

```diff
@@ -30,6 +31,13 @@ def old_profiles
           ref_id: @op_profiles.map(&:id), benchmark: @benchmark&.id
         ).index_by(&:ref_id)
       end
+
+      def value_overrides(op_profile)
+        op_profile.refined_values.each_with_object({}) do |(value_id, selector), value_map|
+          value_definition = @op_value_definitions.find { |v| v.id == value_id }
```

Comment:
This is `O(n*m)` and you could do this with `O(n*1)` if you follow the same pattern that we already have in other places with `index_by`.

Reply:
And I see that you already do the indexing below...please use it.